components/browser/menu2/src/main/java/mozilla/components/browser/menu2/BrowserMenuController.kt

```diff
@@ -43,7 +43,8 @@ class BrowserMenuController(
      */
     override fun show(
         anchor: View,
-        orientation: Orientation?
+        orientation: Orientation?,
+        forceOrientation: Boolean,
```

Comment:
Drive by - you should add a @param for this

Reply:
My mistake, I'll add @param for all of them.  Thanks